distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -936,21 +923,9 @@ type Integer
         ## It's already an integer so unless decimal_places is
            negative, the value is unchanged.
         if decimal_places >= 0 then self else
-            Rounding_Helpers.check_decimal_places decimal_places <| Rounding_Helpers.check_round_input self <|
-                scale = 10 ^ -decimal_places
-                halfway = scale.div 2
-                remainder = self % scale
-                scaled_down = self.div scale
-                result_unnudged = scaled_down * scale
-                case self >= 0 of
-                    True ->
-                        half_goes_up = if use_bankers then (scaled_down % 2) != 0 else self >= 0
-                        round_up = if half_goes_up then remainder >= halfway else remainder > halfway
-                        if round_up  then result_unnudged + scale else result_unnudged
-                    False ->
-                        half_goes_up = if use_bankers then (scaled_down % 2) == 0 else self >= 0
-                        round_up = if half_goes_up then remainder < -halfway else remainder <= -halfway
-                        if round_up then result_unnudged - scale else result_unnudged
+            # Don't check decimal_places here, it's checked in the Java.
+            Rounding_Helpers.check_round_input self <|
+                Math_Utils.roundLong self decimal_places use_bankers
```

Comment:
`roundLong` may throw a `IllegalArgumentException`, but I don't see it being caught and converted to Enso error.

Reply:
I reverted to the Enso integer round, since it was faster. So this is no longer used.